### ISSUE TITLE
Daily recurring event with no DTEND results in warning

### ISSUE THREAD

**Issue by wavexx** [pimutils/khal#539]
I've created a full-day event which repeats daily in the android calendar which has no ending date.
ikhal complains with the following:

````
warning: 703922f2-7514-44ec-bed1-3462d84e587a.ics ("Th75"): The event's end date property contains    
the same value as the start date, which is invalid as per RFC 2445. Khal will assume this is meant to 
be single-day event on 2016-12-02
````

I checked the ics, but there's no DTEND property, which seems correct as per RFC. I assume internally ikhal fills this value and then complains?

The main problem is that it also writes this info over the ikhal interface when dealing with the event, which completely destroys the usability of the interface. The event duration and repetition itself however seems to be correctly handled.

The exported ics contains:

````
BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//CALENDARSERVER.ORG//NONSGML Version 1//EN
BEGIN:VEVENT
SUMMARY:Th75
DTSTART;VALUE=DATE:20161119
DURATION:P1D
DTSTAMP:20161120T180134Z
UID:5b9226dc-d1a8-4a40-a190-45c5f904c367
RRULE:FREQ=DAILY;INTERVAL=5;WKST=MO
STATUS:CONFIRMED
TRANSP:TRANSPARENT
END:VEVENT
END:VCALENDAR
````

**Comment by geier** [comment]
Yeah, sounds like a bug. Reminder to myself: fix together with #444.

**Comment by geier** [comment]
The errors should no longer be shown over the screen (in master).
